### PR TITLE
Issue 411: Updating Pravega configuration key names

### LINKED ISSUE #411: Use new Pravega configuration key names in code and documentation
### Description
Pravega [pull request 4713](https://github.com/pravega/pravega/pull/4713) modified configuration key across the board. While the names are backward compatible, we should be using the new names by default now. Accordingly, code and documentation need to be modified for the operators. 

For a mapping of old to new names, see [this](https://github.com/pravega/pravega/wiki/Configuration-Name-Changes). 

From an upgrade perspective, keep in mind that if both new and old names are specified, the old ones take effect. 

### Importance
Should-have

### PR DESCRIPTION
**Pull request by SrishT**
Signed-off-by: SrishT <Srishti.Thakkar@dell.com>

### Change log description
Modifying the configuration key names mentioned in the operator documentation and sample manifests in accordance with the changes made by [this PR](https://github.com/pravega/pravega/pull/4713) on Pravega side.

### Purpose of the change
Fixes #411

### CODE REVIEW

**Comment by SrishT** [comment]
> Please change field `pravegaservice.cacheMaxSize` in `upgrade-cluster.md ` and `bookkeeper.bkAckQuorumSize` in charts/pravega/values.yaml

made the change

File: doc/longtermstorage.md [hunk 1]
```diff
@@ -133,7 +133,7 @@ Pravega can also use an S3-compatible storage backend such as [Dell EMC ECS](htt
     ```
     ...
     spec:
-    longtermStorage:
+      longtermStorage:
```
Comment (ravisharda) [inline doc/longtermstorage.md:136]: Why does this line show as changed? It looks exactly the same. Please revert the change if it wasn't intended.

File: doc/longtermstorage.md [hunk 2]
```diff
@@ -169,7 +169,7 @@ Refer to the steps below to add ECS server certificate or CA's certificate into
     $ kubectl create -f ecs-tls.yaml
     ```
 
-3. In Pravega manifest, add the secret name defined above into "tls/static/caBundle" section. 
+3. In Pravega manifest, add the secret name defined above into "tls/static/caBundle" section.
```
Comment (ravisharda) [inline doc/longtermstorage.md:172]: This line shows as changed but it has exactly the same content as earlier. Please revert the change.

File: doc/minikube_setup.md [hunk 3]
```diff
@@ -44,26 +44,26 @@ Create a single node Bookkeeper Cluster using the [BookKeeper Operator](https://
...
-          requests:
-            storage: 10Gi
-
-      indexVolumeClaimTemplate:
-        accessModes: [ "ReadWriteOnce" ]
-        storageClassName: "standard"
-        resources:
-          requests:
-            storage: 10Gi
+    ledgerVolumeClaimTemplate:
```
Comment (ravisharda) [inline doc/minikube_setup.md:47]: I don't see any change in these lines 47 to 66. They have exactly the same content as earlier. Please revert the change.

File: example/cr-detailed.yaml [hunk 4]
```diff
@@ -61,26 +61,25 @@ spec:
...
-      pravegaservice.certFile: "/etc/secret-volume/segmentStore01.pem"
-      pravegaservice.keyFile: "/etc/secret-volume/segmentStore01.key.pem"
+      controller.security.tls.enable: "true"
+      controller.security.tls.server.certificate.location: "/etc/secret-volume/controller01.pem"
+      controller.security.tls.server.privateKey.location: "/etc/secret-volume/controller01.key.pem"
+      controller.security.tls.server.keyStore.location: "/etc/secret-volume/controller01.jks"
+      controller.security.tls.server.keyStore.pwd.location: "/etc/secret-volume/password"
+      pravegaservice.security.tls.enable: "true"
+      pravegaservice.security.tls.server.certificate.location: "/etc/secret-volume/segmentStore01.pem"
+      pravegaservice.security.tls.server.privateKey.location: "/etc/secret-volume/segmentStore01.key.pem"
```
Comment (ravisharda) [inline example/cr-detailed.yaml:83]: You might want to add `auth` related params as well in here.

File: doc/minikube_setup.md (hunk 3 above)
Comment (SrishT) [reply doc/minikube_setup.md:47]: This was an intentional change to correct the indentation of these lines

File: doc/longtermstorage.md (hunk 1 above)
Comment (SrishT) [reply doc/longtermstorage.md:136]: This was an intentional change in indentation

File: doc/longtermstorage.md (hunk 2 above)
Comment (SrishT) [reply doc/longtermstorage.md]: done

File: example/cr-detailed.yaml (hunk 4 above)
Comment (SrishT) [reply example/cr-detailed.yaml:91]: done
